QWeb/keywords/input_.py

```diff
@@ -683,10 +685,20 @@ def press_key(locator, key, anchor="1", timeout='0', **kwargs):
     ----------------
     \`TypeSecret\`, \`TypeText\`, \`WriteText\`
     """
+    driver = browser.return_browser()
+    action = ActionChains(driver)
     try:
         input_element = input_.get_input_elements_from_all_documents(
             locator, anchor, timeout=timeout, index=1, **kwargs)
         key = input_handler.check_key(key)
-        input_element.send_keys(key)
+
+        if key[0] == '\ue03d' and util.is_safari():  # COMMAND key workaround on safari
+            javascript.execute_javascript("arguments[0].focus();", input_element)
+            action.key_down(key[0]) \
+                  .send_keys(key[1]) \
+                  .key_up(key[0]).perform()
```

Comment:
cmd alone will not make any sense, but we can just send it as any other key. Empty key list should not be possible, since then the "key" argument is not given. I'll check for length as well in CMD cases, we will support only "normal" cases that make sense in text field context for now (CMD+C, CMD+A etc.).

Reply:
New commit should resolve this